etc/scripts/genlicspdx.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
```

Comment:
@johnmhoran you mention some interest for the SPDX script... this is it.

Reply:
Thank you @pombredanne .  👍